Package.swift

```diff
@@ -17,18 +17,20 @@ import PackageDescription
 
 let package = Package(
     name: "async-http-client",
+    platforms: [.iOS(.v12), .tvOS(.v12)],
     products: [
         .library(name: "AsyncHTTPClient", targets: ["AsyncHTTPClient"]),
     ],
     dependencies: [
         .package(url: "https://github.com/apple/swift-nio.git", from: "2.13.1"),
         .package(url: "https://github.com/apple/swift-nio-ssl.git", from: "2.4.1"),
```

Comment:
we need to update `nio-ssl` version to `2.7.0` to get `NIOSSLClientTLSProvider`

Reply:
done